HACKING.md

```diff
@@ -0,0 +1,150 @@
+# Ceph ROCK Hacking Guide
+The aim of this document is to enable new developers/users get familiarised with the build tools and Ceph ROCK repository so that they can build and contribute to [Ceph-Containers](https://github.com/canonical/ceph-containers).
+
+## Table of Contents
+1. [Introduction to ROCKs](#introduction-to-rocks)
+2. [Tools](#tools)
+3. [References](#references)
+4. [Build Guide](#build-guide)
+6. [Uisng Ceph ROCKs](#using-ceph-rocks)
+
+## Introduction to ROCKs
+> A [ROCK](https://canonical-rockcraft.readthedocs-hosted.com/en/latest/explanation/rocks/) is an OCI compliant container image archive built on top of Ubuntu LTS releases. ROCKs are interoperable with OCI-compliant tools and platforms.
```

Comment:
Maybe here as well a link to explain "OCI", e.g. https://en.wikipedia.org/wiki/Open_Container_Initiative